src/delay.cpp

```diff
@@ -88,7 +88,7 @@ class DelayLine : public CMT_PluginInstance {
 
   DelayLine(const unsigned long lSampleRate,
 	    const LADSPA_Data fMaximumDelay) 
-    : CMT_PluginInstance(4),
+    : CMT_PluginInstance(5),
```

Comment:
I would rather convert to `enum` and then use an `n_ports` enum (like in sledgehammer.cpp) in order to elliminate a constant, but I can also understand the argument of wanting a minimal change.

Reply:
I agree that the `enum` approach is better, but I'm inclined to go for the minimal fix here. There seems to be a preference for the plugins using an enum to be in their own namespace, and the macro constants would need to be replaced with enumerators too. This is more than I was planning to do here, which is just an incidental change to fix this bug while I'm focusing on the MSVC support.